src/app/config/actions.js

```diff
@@ -323,3 +323,22 @@ export const updateCollectionSuccess = collection => ({
     type: types.UPDATE_COLLECTION_SUCCESS,
     collection,
 });
+
+export function createUserSuccess(user) {
```

Comment:
I decided to add logic with the user here as it is more related to the object users than the state user I created before. this need refactoring as state user is more about authentication not app users